### ISSUE TITLE
yarn generate returns an error

### ISSUE THREAD

**Issue by Tch4lla** [ThePrimeagen/kata-machine#12]
After forking, cloning, and following the guide in the readme, when getting to the yarn generate step this is returned 

`$ yarn generate
yarn run v1.22.19
$ ./scripts/generate
'.\scripts\generate' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

Is there something missing in the readme ? 

yarn version 1.22.19
node version 16.5.0

**Comment by jawwadHussain302** [comment]
```
'.\scripts\generate' is not recognized as an internal or external command,
operable program or batch file.
```
same error

**Comment by ronakmehtav** [comment]
you can use any package manager you wish not just yarn, I my self am using pnpm.

**Comment by MustafaAliDS** [comment]
I am tried using npm as well, but i still received an error

**Comment by AhmadS7** [comment]
I was getting the same error on Windows 11, but when I tried it with WSL, it worked perfectly.

**Comment by ThePrimeagen** [comment]
i did intend this to be used with WSL

**Comment by ThePrimeagen** [comment]
this may have been fixed with #11
